src/threadpool.c

```diff
@@ -170,11 +170,6 @@ void uv__threadpool_cleanup(void) {
   if (nthreads == 0)
     return;
 
-#ifndef __MVS__
-  /* TODO(gabylb) - zos: revisit when Woz compiler is available. */
-  post(&exit_message, UV__WORK_CPU);
-#endif
-
```

Comment:
This is not dead code -- it's for platforms that are _not_ z/OS.

Reply:
Whoops! Nevermind I just misread some documentation.